src/validator/collator.rs

```diff
@@ -262,6 +262,7 @@ struct CollatorData {
     out_msg_count: usize,
     in_msg_count: usize,
     remove_count: usize,
+    depth_total: usize,
```

Comment:
Consider using not so general name like 'msg_queue_total_changed_depth'. And it is not a "metric to report statsd"

Reply:
resolved